examples/classic/kubernetes-private.classic.json

```diff
@@ -0,0 +1,38 @@
+{
+  "apiVersion": "vlabs",
+  "properties": {
+    "orchestratorProfile": {
+      "orchestratorType": "Kubernetes",
+      "kubernetesConfig": {

```

Comment:
Yeah that's a great point. I had originally thought I could use the FQDN to reference the ILB but that doesn't appear to be the case. I've got a bunch of questions about how this should be implemented in the issue #221. Please feel free to let me know the best way to proceed.